### PR TITLE
Upgrade to Quarkus 3.0.3.Final

### LINKED ISSUE #20300: Upgrade to Quarkus 3.0.3.Final
### Description

Upgrade to Quarkus 3.0.3.Final

### PR DESCRIPTION
**Pull request by vmuzikar**
Closes #20300

### CODE REVIEW

**Review by abstractj — DISMISSED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -82,7 +82,7 @@
         <h2.version>2.1.214</h2.version>
         <hibernate-orm.version>6.2.1.Final</hibernate-orm.version>
         <hibernate.c3p0.version>${hibernate-orm.version}</hibernate.c3p0.version>
-        <infinispan.version>14.0.8.Final</infinispan.version>
+        <infinispan.version>14.0.9.Final</infinispan.version>
```
Comment (hmlnarik) [inline pom.xml:85]: @martin-kanis @mhajas This change looks safe to me but I'd like to have your opinion here.

Comment (martin-kanis) [reply pom.xml:85]: LGTM

**Comment by ASzc** [comment]
Should be ok for product. Still waiting for RHBQ 3 before we can narrow down a Quarkus version

**Review by hmlnarik — DISMISSED** [review]
Approving per @martin-kanis 's approval

**Comment by vmuzikar** [comment]
Sorry, had to rebase to resolve conflicts. Could you please re-review?

**Review by abstractj — APPROVED** [review]

**Comment by abstractj** [comment]
@vmuzikar enabled the auto-merge based on previous approvals.

**Pull request merged**